### PR TITLE
enter text code

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers 

Do 'already on keybase? login'
Connect another device
Phone
Type text code instead
screen shot 2016-02-22 at 2 12 26 pm

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers  bump

File: shared/login/register/code-page/index.render.desktop.js [hunk 1]
```diff
@@ -54,39 +57,32 @@ export default class CodePageRender extends Component {
...
           onChange={event => this.props.onChangeText(event.target.value)}
         />
-        <RaisedButton
-          style={{alignSelf: 'flex-end', marginTop: 20}}
-          label='Submit'
-          primary
-          onClick={() => this.props.textEntered(codePageModeEnterText)}
-        />
-      </div>
+        <Button dz2 style={{alignSelf: 'flex-end', marginTop: 35, marginBottom: 20}} label='Continue' onClick={() => this.props.textEntered(codePageModeEnterText)} />
```
Comment (cjb) [inline shared/login/register/code-page/index.render.desktop.js]: Ah, now that dz2 Button's been merged into master, I'd expect this to fail Flow 'cause you don't have a `type='Primary'` or similar specified.  Try again on master?

**Comment by cjb** [comment]
:thumbsup: %C

File: shared/login/register/code-page/index.render.desktop.js (hunk 1 above)
Comment (chrisnojima) [inline shared/login/register/code-page/index.render.desktop.js]: lemme rebase and reverify it looks good

Comment (chrisnojima) [inline shared/login/register/code-page/index.render.desktop.js]: actually give me a thumb and i'll merge this into the base branch, then rebase that thing

**Pull request merged**